torch/csrc/Stream.cpp

```diff
@@ -12,10 +12,20 @@ static PyObject* THPStream_pynew(
     PyObject* args,
     PyObject* kwargs) {
   HANDLE_TH_ERRORS
-  uint64_t cdata = 0;
+  int64_t stream_id = 0;
+  int64_t device_index = 0;
+  int64_t device_type = 0;
   // NOLINTNEXTLINE(cppcoreguidelines-avoid-c-arrays,modernize-avoid-c-arrays,clang-diagnostic-writable-strings)
-  static char* kwlist[] = {"_cdata", nullptr};
-  if (!PyArg_ParseTupleAndKeywords(args, kwargs, "|K", kwlist, &cdata)) {
+  static char* kwlist[] = {
+      "_stream_id", "_device_index", "_device_type", nullptr};
+  if (!PyArg_ParseTupleAndKeywords(
```

Comment:
It seems that this API is user-accessible? TODO: attempt to match behavior from passing a single `int64_t` as much as possible.

Reply:
I don't think we document that you can pass a `_cdata` in directly. Think it's OK to break this